src/test/java/jssc/SerialNativeInterfaceTest.java

```diff
@@ -50,4 +59,16 @@ public void reportsWriteErrorsAsIOException() throws Exception {
         testTarget.writeBytes(fd, buf);
     }
 
+    @Test
+    public void throwsIllegalArgumentExceptionIfPortHandleIllegal() throws Exception {
+        assumeFalse(SerialNativeInterface.getOsType() == SerialNativeInterface.OS_WINDOWS);
```

Comment:
Now that you've implemented the exception in `windows/jssc.cpp`'s readBytes, should we let this test run?